.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
What is this for exactly?

Reply:
It checks out the code with LF line endings on Windows instead of the default CRLF. The CRLF line endings seemed to cause some issues when running Prettier on CI.